packages/jobs/lib/crons/deleteSyncsData.ts

```diff
@@ -55,12 +58,10 @@ export async function exec(): Promise<void> {
 
             // -----
             // Soft delete schedules
-            let countSchedules = 0;
-            do {
-                countSchedules = await softDeleteSchedules({ syncId: sync.id, limit: limitSchedules });
-                logger.info(`[deleteSyncs] soft deleted ${countSchedules} schedules`);
-                metrics.increment(metrics.Types.JOBS_DELETE_SYNCS_DATA_SCHEDULES, countSchedules);
-            } while (countSchedules >= limitSchedules);
+            const resSchedule = await orchestrator.deleteSync({ syncId: sync.id, environmentId: sync.environmentId });
+            const deletedScheduleCount = resSchedule.isErr() ? 1 : 0;
+            logger.info(`[deleteSyncs] soft deleted ${deletedScheduleCount} schedules`);
+            metrics.increment(metrics.Types.JOBS_DELETE_SYNCS_DATA_SCHEDULES, deletedScheduleCount);
 
```

Comment:
forgot this cron job. I will cleanup by hand the schedules that we are currently not deleting